CMakeLists.txt

```diff
@@ -136,6 +137,17 @@ if (AMR_WIND_ENABLE_OPENFAST)
   target_link_libraries(${amr_wind_lib_name} PUBLIC ${OpenFAST_LIBRARIES})
 endif()
 
+if(AMR_WIND_ENABLE_ASCENT)
+  set(CMAKE_PREFIX_PATH ${ASCENT_DIR} ${CMAKE_PREFIX_PATH})
+  find_package(Ascent QUIET REQUIRED)
+  if (Ascent_FOUND)
+    message(STATUS "Found Ascent = ${ASCENT_INCLUDE_DIRS}")
+  endif()
+  target_include_directories(${amr_wind_lib_name} PUBLIC ${ASCENT_INCLUDE_DIRS})
+  target_link_libraries(${amr_wind_lib_name} PUBLIC ${ASCENT_LIBRARIES})
+  target_compile_definitions(${amr_wind_lib_name} PRIVATE AMR_WIND_USE_ASCENT)
```

Comment:
Does ascent provide the modern CMake library definitions?

Reply:
Looks like they do; I'll update this.